src/features/workflow/WorkflowCellControls.tsx

```diff
@@ -2,23 +2,25 @@ import React from "react";
 import { useDispatch } from "react-redux";
 
 import {
-  clearOutputWorkflowTransformStep,
   duplicateWorkflowTransformStep,
   moveWorkflowTransformStepDown,
   moveWorkflowTransformStepUp,
   removeWorkflowTransformStep
 } from './state/workflowActions';
 import WorkflowCellControlButton from "./WorkflowCellControlButton";
+import { removeEvent } from "../events/state/eventsActions";
 
 
 interface WorkflowCellControlsProps {
   index: number
   setAnimatedCell: (i:number) => void
+  sessionId: string | undefined
```

Comment:
let's force this to only accept a `string`, rather than `string | undefined` & make whatever changes to the call site so that we are always passing in a string.